### ISSUE TITLE
settings gets redirected to personalSettings

### ISSUE THREAD

**Issue by xsteadfastx** [airsonic/airsonic#1722]
## Problem description

Trying to access settings gets redirected to personalSettings URL. For some time i had the problem that the "settings" button was missing. No big problem, because i could access it entering the settings URL like `https://foo.bar.tld/settings`. Now doing that i get redirected to `https://foo.bar.tld/personalSettings`. So i cant find a way to trigger a library rescan manually.

## System information

 * **Airsonic version**: 10.6.2-RELEASE – May 3, 2020
 * **Operating system**: official docker image on a ubuntu host
 * **Java version**: java 1.8.0_242
 * **Client**: firefox and chrome

**Comment by xsteadfastx** [comment]
i try to access `/generalSettings.view` and get a 403 error.

**Comment by tesshucom** [comment]
Looks like a derivative of #1470.

**Comment by xsteadfastx** [comment]
yes i think so too... i recreated the admin with id 1 and everything worked again.

**Comment by tesshucom** [comment]
#1470 is the root, and #1131 increased the frequency of occurrence.
